cmd/scheduler/app/options/options.go

```diff
@@ -156,6 +168,8 @@ func (s *ServerOption) AddFlags(fs *pflag.FlagSet) {
 	fs.Uint32Var(&s.NodeWorkerThreads, "node-worker-threads", defaultNodeWorkers, "The number of threads syncing node operations.")
 	fs.StringSliceVar(&s.IgnoredCSIProvisioners, "ignored-provisioners", nil, "The provisioners that will be ignored during pod pvc request computation and preemption.")
 	fs.BoolVar(&s.DisableDefaultSchedulerConfig, "disable-default-scheduler-config", false, "The flag indicates whether the scheduler should avoid using the default configuration if the provided scheduler configuration is invalid.")
+	fs.StringVar(&s.ShardingMode, "scheduler-sharding-mode", util.NoneShardingMode, "The node sharding mode for scheduling")
+	fs.StringVar(&s.ShardName, "scheduler-sharding-name", defaultSchedulerName, "The name of shard used for this scheduler")
```

Comment:
maybe we donot need a flag to indicate the name of nodeshard, just a const. 
Should not use scheduler name, even they have same value, which is just confusing